packages/host/app/components/matrix/room-message.gts

```diff
@@ -112,6 +116,19 @@ export default class RoomMessage extends Component<Signature> {
     return this.commandService.run.unlinked().perform(command);
   });
 
+  private get formattedMessage() {
+    if (
+      !this.message.formattedMessage ||
+      isHtml(this.message.formattedMessage)
+    ) {
+      return this.message.formattedMessage;
+    }
+    return markdownToHtml(
```

Comment:
The formatted message must be already in html. However, this handle the old cases where ai bot sent formatted message in markdown.